[ISSUE #9930] Solve the problem of duplicate assignment in the init method of the GrpcMessagingApplication class

709572460 opened the pull request:
Solve the problem of duplicate assignment in the init method of the GrpcMessagingApplication class

709572460 committed:
fix(proxy): Solve the problem of duplicate assignment in the init method of the GrpcMessagingApplication class

Solve the problem of duplicate assignment in the init method of the GrpcMessagingApplication class.

Closes #9930

lizhimins commented on proxy/src/main/java/org/apache/rocketmq/proxy/grpc/v2/GrpcMessagingApplication.java:
```diff
@@ -136,7 +136,10 @@ protected GrpcMessagingApplication(GrpcMessagingActivity grpcMessagingActivity,
-        this.routeThreadPoolExecutor.setRejectedExecutionHandler(rejectedExecutionHandler);
+
+        //Repeated assignment
+        //this.routeThreadPoolExecutor.setRejectedExecutionHandler(rejectedExecutionHandler);
```
You can directly remove the duplicate parts.

709572460 replied:
OK

lizhimins reviewed (commented):
You can directly remove the duplicate parts.

709572460 committed:
fix(proxy): Solve the problem of duplicate assignment in the init method of the GrpcMessagingApplication class

Solve the problem of duplicate assignment in the init method of the GrpcMessagingApplication class.
removed the duplicate parts.

Closes #9930

709572460 commented:
I've deleted all the redundant code

709572460 committed:
Solve the problem of duplicate assignment in the init method of the GrpcMessagingApplication class.
removed the duplicate parts.
